CRM/Upgrade/Incremental/sql/5.51.alpha1.mysql.tpl

```diff
@@ -1 +1,2 @@
 {* file to handle db changes in 5.51.alpha1 during upgrade *}
+ALTER TABLE `civicrm_mailing_bounce_type` CHANGE `name` `name` VARCHAR(256) CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL COMMENT 'Type of bounce', CHANGE `description` `description` VARCHAR(2048) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL DEFAULT NULL COMMENT 'A description of this bounce type';
```

Comment:
Just noting that this may not be 100% right especially for those that have converted to utf8mb4

Reply:
Thank you, makes sense. I have updated the PR